Cargo.toml

```diff
@@ -64,7 +64,10 @@ name = "simple_listener"
 [features]
 # If you add a feature here don't forget to add it to the test workflow in workflows/rust.yml!
 default = ["transport-tcp", "transport-tcp-native-tls"]
-refreshing-token = ["reqwest", "with-serde"]
+refreshing-token = ["reqwest/default-tls", "__refreshing-token"]
+refreshing-token-rustls-native-roots = ["reqwest/rustls-tls-native-roots", "__refreshing-token"]
+refreshing-token-rustls-webpki-roots = ["reqwest/rustls-tls-webpki-roots", "__refreshing-token"]
+__refreshing-token = ["with-serde"]
```

Comment:
I'd prefer if we renamed the previously existing `refreshing-token` to `refreshing-token-native-tls`. I already have a breaking/major release coming up, so this would be a good time to ensure all features are named consistently, at the cost of breaking backwards compatibility.